2017-09-08 17:26:04,781 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:26:20,668 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:27:08,605 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:28:41,327 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:28:53,050 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:29:19,372 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:29:49,172 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:30:04,001 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:31:04,624 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:31:15,533 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:31:38,502 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,550 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,554 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,555 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,555 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,556 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:31:40,557 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:21,776 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:32:23,834 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:16,109 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,162 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:33:36,161 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:00,377 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:35:38,605 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,653 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:35:39,652 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:83)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:29,782 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,830 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,830 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:38:30,829 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:84)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:14,156 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,214 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:43:15,228 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:18,835 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:44:28,882 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,883 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,883 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,883 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,883 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,883 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,883 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,882 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,894 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,894 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:44:28,894 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码校验时发生异常>
java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.lanxi.consumeLoan.service.CheckService.isPhoneNew(CheckService.java:85)
	at com.lanxi.consumeLoan.test.Test20170815.lambda$4(Test20170815.java:52)
	at java.lang.Thread.run(Unknown Source)
2017-09-08 17:51:06,348 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:51:59,261 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:52:00,001 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,003 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,004 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,005 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,008 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,009 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,012 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,012 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,013 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,505 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,517 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,528 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,528 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,532 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,540 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:00,540 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:01,510 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:01,513 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:02,529 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:03,519 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,912 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,913 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,917 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,918 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,919 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,920 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,925 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,928 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,930 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,931 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:19,932 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:20,496 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:20,499 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:20,500 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:20,505 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:21,496 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:21,498 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:22,491 INFO [com.lanxi.consumeLoan.service.CheckService] - <手机号码:15068610940
归属地  :浙江台州市
营运商  :移动
卡类型  :移动150卡
区号    :0576
邮编    :318000>
2017-09-08 17:52:49,742 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:56:46,605 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:57:06,537 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:57:19,927 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 17:58:13,428 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 18:00:27,542 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 18:02:09,784 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 18:02:15,243 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 18:02:21,174 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
2017-09-08 18:03:00,673 INFO [com.lanxi.util.utils.LoggerUtil] - <log4j初始化完成!>
